src/main/java/com/theironyard/repositories/BookRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.theironyard.repositories;
+
+import com.theironyard.models.Book;
+import com.theironyard.models.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+/**
+ * Created by melmo on 12/29/16.
+ */
+public interface BookRepository extends JpaRepository<Book, Integer>{
```

Comment:
As mentioned in the controller, there are ways to pass a sort object into the methods so that you don't have 3 methods that are the same except the ordering.